dashboard/app/models/course_offering.rb

```diff
@@ -445,6 +445,10 @@ def high_school_level?
     grade_levels_list.any? {|g| HIGH_SCHOOL_GRADES.include?(g)}
   end
 
+  def self_paced_pl?
```

Comment:
Should we instead do this based on participant audience? Anything that has participant as teacher or facilitator. I worry that this category is more fragile and having two sources of truth is not really ideal

Reply:
in course_types.rb we have an exist pl_course? method. Wonder if we can leverage that here

Reply:
As long as it's okay that all units with a participant audience of teacher or facilitator have this functionality, I'm happy to make that change. Agree it would be less fragile

Reply:
Yup! Its good. Thank you!